### ISSUE TITLE
Ambush missions on non-abandoned plots generate with extra parcel when Mission Overhaul: Core is loaded

### ISSUE THREAD

**Issue by h4ilst0rm** [WOTCStrategyOverhaul/CovertInfiltration#558]
![XCOM 2 Screenshot 2020 07 03 - 09 38 02 54](https://user-images.githubusercontent.com/59488828/86441730-58ccc880-bd15-11ea-8708-45a9baa89508.png)
![XCOM 2 Screenshot 2020 07 03 - 09 38 09 82](https://user-images.githubusercontent.com/59488828/86441733-5a968c00-bd15-11ea-93d2-6d989c0729ee.png)
![XCOM 2 Screenshot 2020 07 03 - 09 44 14 71](https://user-images.githubusercontent.com/59488828/86441769-671ae480-bd15-11ea-9d4a-49fe407654b9.png)
![XCOM 2 Screenshot 2020 07 03 - 09 44 29 98](https://user-images.githubusercontent.com/59488828/86441773-684c1180-bd15-11ea-84c5-b2bdaa5e4655.png)
![XCOM 2 Screenshot 2020 07 03 - 09 47 06 39](https://user-images.githubusercontent.com/59488828/86441782-6bdf9880-bd15-11ea-8fec-f13fd2e80505.png)
![XCOM 2 Screenshot 2020 07 03 - 09 47 12 95](https://user-images.githubusercontent.com/59488828/86441791-6f731f80-bd15-11ea-921a-52ec867aafae.png)

**Comment by Xymanek** [comment]
Do you use any map packs? I recall testing all plots for that "hack" and never saw this issue, neither did any YT campaign so far...

**Comment by h4ilst0rm** [comment]
I do use a few map packs, though they were all disabled when testing this in skirmish mode.
I did forget to clean configs, but just now did that and got the best one yet:

![XCOM 2 Screenshot 2020 07 03 - 12 15 35 98](https://user-images.githubusercontent.com/59488828/86453966-4491c700-bd27-11ea-80a1-189c7e1cb75a.png)
![XCOM 2 Screenshot 2020 07 03 - 12 15 42 60](https://user-images.githubusercontent.com/59488828/86453974-46f42100-bd27-11ea-9188-f015ac2f2072.png)

full modlist used:
```
Unsorted (2):
	TLP: Unlocker                                 	TLP_UnlockEverything       	1535023259 	
	Skirmish Mode+                                	AllEquipmentsinSkirmish    	1535761480 	

CI core (13):
	WotC: robojumper's Squad Select               	robojumperSquadSelect_WotC 	1122974240 	
	Better Second Wave Mod Support                	BetterSecondWaveSupport    	1149493976 	
	[WOTC] Open Squad Select at any time          	SquadSelectAtAnyTime       	1529472981 	
	Highlander for CI Beta 3                      	X2WOTCCommunityHighlander  	2122546411 	
	Accurate Situation Report Beta                	AccurateSituationReport    	2122561731 	
	Living Space Beta                             	LivingSpace                	2122563289 	
	Infiltrator's Disguises Beta                  	InfiltratorsDisguises      	2126774922 	
	Prototype Armoury Beta                        	PrototypeArmoury           	2126777138 	
	Covert Infiltration Beta 3                    	CovertInfiltration         	2135314482 	
	[WotC] Mod Config Menu                        	ModConfigMenu              	667104300  	
	Pyrrhic Victories                             	CatastrophicVictories      	1531883724 	
	[WotC] Mission Overhaul: Core                 	WotC_MissionOverhaul_Core  	1877944427 	
	Alien Hunters Highlander Plugin for CI Beta 3 	DLC2CommunityHighlander    	2126773442 	
```

**Comment by Xymanek** [comment]
I don't have this issue on that plot:

![image](https://user-images.githubusercontent.com/2865341/86566928-26b8a200-bf73-11ea-87b9-515f5491b10a.png)

However, looking at your screenshot, the objective parcel is overlapping 2 other parcels (???), so I feel like the issue is not with the specific plot. Can you repro the same issue on all plots reliably?

**Comment by h4ilst0rm** [comment]
Yeah it is really strange how it seems to try and add an extra parcel and when there is no place for it the whole thing breaks.

I bit the bullet and verified files only for the issue to persist. With some additional fiddling around I found the culprit:
Mission Overhaul: Core.

No idea as to how or why, but with overhaul active the extra parcel is present on all non-abandoned plot types.
Quick check into overhauls configs + scripts didn't give me any insights, though there are some .umap files under /content/maps that could be related to this, though I have no idea what they are used for.

Tested also with map packs enabled which seemed to work as expected when not running overhaul (few tries in skirmish).
